src/recipes/muskmelon-panaka.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "<Muskmelon Panaka>",
```

Comment:
```suggestion
    "name": "Muskmelon Panaka",
```

You can remove these from the file. They are just to indicate it's a replacement token.